### DISCUSSION TITLE
Escaping curly braces in .po files

### DISCUSSION THREAD

**Discussion by xantinium** [lingui/js-lingui#1373] [Q&A] [answered]
Is there a way to have a curly braces in .po file without recognizing them as a special symbol?
For example, I wanna add smth like this:
"This is a {test} message"
In this case "test" is a variable, but I want it to be a plain text. Can I add some chars to escape these curly braces? I mean:
compile("This is a {test} message") // "This is a {test} message"

**Comment by timofei-iatsenko** [comment, marked as answer]
Use single quotes for escaping. 
```"This is a '{test}' message"```

https://unicode-org.github.io/icu/userguide/format_parse/messages/#quotingescaping

**Comment by xantinium** [comment]
I tried it, but it didn't work. Instead of escaping, the characters simply disappeared.
I mean: `This is a '{test}' message` -> `This is a message`
I've tested a lot and escaping working perfectly inside JS function, but not in a .po files.

**Reply by timofei-iatsenko** [reply]
You should try this is in the latest version. It was fixed recently

**Comment by xantinium** [comment]
Thanks a lot :D
